### PR TITLE
Refits/Rebrands Chemistry Module

### PR DESCRIPTION
**Pull request by Wisemonster**
[SILICONS] [QOL]

## About the PR 
This pr rebrands the chemistry cyborg module to the research module, and gives the module a pen, artifact form bin, and gps.

## Why's this needed? 
This lets silicons help with the other sub departments in science, rather than restricting them to chemistry.

## Changelog 

```changelog
(u)Wisemonster
(*)The chemistry cyborg module has been refitted to the research module
(+)The chemistry (now research) module now comes with a pen, artifact analysis from bin, and gps.
```

### CODE REVIEW

**Review by ZeWaka — COMMENTED** [review]
Looks good - did you test the artifact form interactions?

**Comment by Wisemonster** [comment]
I did, seems at the moment borgs can't actually put the forms onto anything, although I was unsure if adding that functionality would've been better suited for another pr. They can also make a lot with no cooldown considering the bins are infinite, although this is also possible with the non-borg versions.

**Comment by Wisemonster** [comment]
I could do one of 2 things to prevent borgs from spamming these, either add a cooldown, or limit how many forms are in their trays.

**Comment by ZeWaka** [comment]
> I could do one of 2 things to prevent borgs from spamming these, either add a cooldown, or limit how many forms are in their trays.

I think a cooldown is probably best. Even just a second would do.

**Comment by Wisemonster** [comment]
Went ahead and added a 1 second delay for the paper bin in the module

File: code/modules/writing/paper.dm [hunk 1]
```diff
@@ -647,6 +647,11 @@
 		n++
 	return "There's [(n > 0) ? n : "no" ] paper[s_es(n)] in \the [src]."
 
+/obj/item/paper_bin/artifact_paper/robot
+	name = "semi-automatice artifact analysis bin"
```
Comment (ZeWaka) [inline code/modules/writing/paper.dm:651]:
```suggestion
	name = "semi-automatic artifact analysis bin"
```

**Review by ZeWaka — APPROVED** [review]

**Review by Tarmunora — APPROVED** [review]

**Pull request merged**